feat(services): Point Slack to Google Drive

catheraaine opened the pull request:
Our Slack documentation has been moved to Google Docs and will be added to the onboarding Trello.
This change encourages inclusiveness, as not ever member of our team is familiar with GitHub.

bengoshow commented on services/README.md:
```diff
@@ -11,6 +11,5 @@ The services we use. Some we pay for, others we mooch off. We love them all.
 * **[Harvest](harvest)**
 * **[InVision](invision)**
 * **[SendGrid](sendgrid)**
-* **[Slack](slack)**
```
Thanks, Cat - would it make sense to still mention that we use Slack, but our docs are in Drive now? Instead of removing the reference altogether?

catheraaine replied:
Oh yeah maybe! A lot of things in this list are very outdated. We'll need a rework in the future, I think it's on @kaseybon's long ToDo list.

catheraaine committed:
feat(services): Point Slack to Google Drive

Our Slack documentation has been moved to Google Docs and will be added to the onboarding Trello.
This change encourages inclusiveness, as not ever member of our team is familiar with GitHub.

bengoshow reviewed (approved):
Sounds great, thanks! Any other updates you want to make before I merge into `main`?
